application/Assignment.java

```diff
@@ -22,7 +22,8 @@ public Assignment(String assignmentName, Class className, int difficulty, Date s
 		this.dueDate = dueDate;
 		this.dueTime = dueTime;
 		this.completed = completed;
-		priority = difficulty * className.getDifficulty();
+		
+		priority = difficulty * className.getDifficulty() * dueDate.compareTo(startDate);
```

Comment:
**TODO:**
also add current date into calculation. possibly remove `dueDate.compareTo(startDate)` or the `startDate` field altogether. @acfranz2